itests/org.jboss.tools.cdk.ui.bot.test/src/org/jboss/tools/cdk/ui/bot/test/server/adapter/openshift/CDKImageRegistryUrlDiscoveryFailureTest.java

```diff
@@ -37,23 +40,35 @@ protected String getServerAdapter() {
 		return SERVER_ADAPTER_32;
 	}
 	
+	@BeforeClass
+	public static void setupCDKImageRegistrUrlDiscovery() {
+		setupOCForWorkspace();
+	}
+	
+	@Before
+	public void setupOpenshiftConnectionWizard() {
+		wizard = getOpenshiftConnectionWizard(findOpenShiftConnection(null, OPENSHIFT_USERNAME));
+		assertTrue(wizard.getAuthSection().getMethod().equals(AuthenticationMethod.BASIC));
+		switchOffPasswordSaving(wizard);
+	}
+	
 	/**
 	 * Covers JBIDE-25049
 	 */
 	@Test
 	public void testRegistryUrlNotFoundDialog() {
 		String shellTitle = OpenShiftLabel.Shell.REGISTRY_URL_NOT_FOUND;
```

Comment:
@odockal I would not use variable for shell title, it could help to have easier readable code